tools/rebasehelpers/util/git.go

```diff
@@ -98,12 +98,15 @@ func (c Commit) GodepsReposChanged() ([]string, error) {
 
 type File string
 
-func (f File) HasGodepsChanges() bool {
+func (f File) HasGodepsChanges(includingGodeps bool) bool {
```

Comment:
Passing in a `bool` just to switch on it inside of the method is bad. You have two different methods, with two different actions, let's not overload `HasGodepsChanges` to try to handle both. Also, the new method signature is very hard to read:

```go
if !file.HasGodepsChanges(false) {
```